src/System.Private.ServiceModel/src/System/IdentityModel/Selectors/SecurityTokenSerializer.cs

```diff
@@ -207,41 +208,5 @@ public virtual bool CanReadKeyIdentifierCore(XmlDictionaryReader reader)
 
             public abstract void WriteKeyIdentifierCore(XmlDictionaryWriter writer, SecurityKeyIdentifier keyIdentifier);
         }
-
-        internal abstract class TokenEntry
```

Comment:
> internal abstract class TokenEntry [](start = 7, length = 35)

Reviewers: the duplicate-and-diverge history between IdentityModel and WCF has really made this merge awkward.  I am attempting to eliminate the internal IdentityModel types that WCF has duplicates for and to converge on the WCF versions.  This deletion is an example.  If you have any caveats to share, I'd like to hear them.  I'd actually prefer to refactor more broadly, but to make this PR reviewable, I'm just trying to eliminate duplication and stick with existing factoring.